definitions/npm/semver_v7.x.x/flow_v0.104.x-/semver_v7.x.x.js

```diff
@@ -0,0 +1,689 @@
+  import type semverCompareBuild from './functions/compare-build';
+  import type semverSort from './functions/sort';
+  import type semverRsort from './functions/rsort';
+  declare export {
+            semverParse as parse,semverValid as valid,semverClean as clean,semverInc as inc,semverDiff as diff,semverMajor as major,semverMinor as minor,semverPatch as patch,semverPrerelease as prerelease,semverCompare as compare,semverRcompare as rcompare,semverCompareLoose as compareLoose,semverCompareBuild as compareBuild,semverSort as sort,semverRsort as rsort
+          }
+  import type semverGt from './functions/gt';
+  import type semverLt from './functions/lt';
+  import type semverEq from './functions/eq';
+  import type semverNeq from './functions/neq';
+  import type semverGte from './functions/gte';
+  import type semverLte from './functions/lte';
+  import type semverCmp from './functions/cmp';
+  import type semverCoerce from './functions/coerce';
+  declare export {
+            semverGt as gt,semverLt as lt,semverEq as eq,semverNeq as neq,semverGte as gte,semverLte as lte,semverCmp as cmp,semverCoerce as coerce
+          }
+  import type semverSatisfies from './functions/satisfies';
+  import type semverMaxSatisfying from './ranges/max-satisfying';
+  import type semverMinSatisfying from './ranges/min-satisfying';
+  import type semverToComparators from './ranges/to-comparators';
+  import type semverMinVersion from './ranges/min-version';
+  import type semverValidRange from './ranges/valid';
+  import type semverOutside from './ranges/outside';
+  import type semverGtr from './ranges/gtr';
+  import type semverLtr from './ranges/ltr';
+  import type semverIntersects from './ranges/intersects';
+  import type simplify from './ranges/simplify';
+  import type rangeSubset from './ranges/subset';
+  declare export {
+            semverSatisfies as satisfies,semverMaxSatisfying as maxSatisfying,semverMinSatisfying as minSatisfying,semverToComparators as toComparators,semverMinVersion as minVersion,semverValidRange as validRange,semverOutside as outside,semverGtr as gtr,semverLtr as ltr,semverIntersects as intersects,simplify as simplifyRange,rangeSubset as subset
+          }
+  import type SemVer from './classes/semver';
+  import type Range from './classes/range';
+  import type Comparator from './classes/comparator';
+  declare export {
+            SemVer,Range,Comparator
+          }
+  import type identifiers from './internals/identifiers';
+  declare export var compareIdentifiers: typeof identifiers.compareIdentifiers;declare export var rcompareIdentifiers: typeof identifiers.rcompareIdentifiers;declare export var SEMVER_SPEC_VERSION: "2.0.0";export type ReleaseType = "major"
+  | "premajor"
+  | "minor"
+  | "preminor"
+  | "patch"
+  | "prepatch"
+  | "prerelease";export interface Options {
+  loose?: boolean,
+  includePrerelease?: boolean,
+  } export type CoerceOptions = {
```

Comment:
Could you make en effort on formatting, run prettier or at least properly separate blocks. This is really harsh to read and not matching our (very minimal) standards.